bindings/rust/wasmedge-sys/src/instance/function.rs

```diff
@@ -415,6 +439,16 @@ impl Function {
         engine.run_func(self, args)
     }
 
+    pub async fn call_async<E: Engine + Send + Sync>(
```

Comment:
Please add doc comments for this new API. As for the format of the API doc comments, please refer to the existed. Thanks!